Source/WebCore/platform/graphics/FontPlatformData.h

```diff
@@ -123,6 +123,18 @@ struct FontPlatformDataAttributes {
         { }
 #endif
 
+#if USE(SKIA)
+    FontPlatformDataAttributes(float size, FontOrientation orientation, FontWidthVariant widthVariant, TextRenderingMode textRenderingMode, bool syntheticBold, bool syntheticOblique, Vector<hb_feature_t>&& features)
```

Comment:
It might be nice to share the common pieces, but this is probably also fine.  This follows an existing pattern.